packages/app/src/components/button/outline/index.tsx

```diff
@@ -125,6 +132,7 @@ const Renewal = function <T = null>({
           'text-xl': size === SIZE.XL,
           'text-2xl': size === SIZE['2XL'],
         },
+        buttonStyle,
```

Comment:
It should probably be inline like
```
{
   'hoge': cs === 'primary',
   'huga': cs !== 'primary'
}
```

because `eslint-plugin-tailwindcss` doesn't check CSS classes outside of `classNames`.

Reply:
Oops, never mind!
eslint-plugin-tailwindcss is not in this app!
I think the way i told is better though.